charts/hedera-mirror-rosetta/values.yaml

```
@@ -223,7 +223,9 @@ terminationGracePeriodSeconds: 60
 
 test:
   enabled: true
-  githubRef: ""
+  github:
+    ref: ""
```

```suggestion
    ref: ""  # Default to Chart app version
```
Note the two spaces before `#` to avoid lint failures.

---

updated, also rename the property to `branch`